big_tests/tests/mod_ping_SUITE.erl

```diff
@@ -155,11 +157,11 @@ disco(Config) ->
 
 ping(ConfigIn) ->
     Domain = domain(),
-    HostType = domain_helper:host_type(mim),
+    HostType = host_type(),
     HostTypePrefix = domain_helper:make_metrics_prefix(HostType),
     Metrics = [
-        {[HostTypePrefix, mod_ping, ping_response],0},
-        {[HostTypePrefix, mod_ping, ping_response_timeout],0}
+        {[HostTypePrefix, mod_ping_response, count], 0},
+        {[HostTypePrefix, mod_ping_response_timeout, count], 0}
```

Comment:
The plan is to gradually remove `mongoose_metrics` - also from tests. Could you replace all such checks with assertions for events?